### ISSUE TITLE
Strange behaviour on Kafka multi consumers

### ISSUE THREAD

**Issue by Gnucki** [php-enqueue/enqueue-dev#470]
Hi,

I'm a bit confused with the use of rdkafka.
I start 2 consumers and a producer with following config:
```yaml
enqueue:
    async_events:
        enabled: false
    transport:
        default: rdkafka
        rdkafka:
            global:
                group.id: '%app.name%'
                offset.store.method: broker
                metadata.broker.list: '%env(KAFKA_BROKER_LIST)%'
            topic:
                offset.store.method: broker
    client: ~
```
I publish a message on a topic with only 1 partition.
I expect only 1 consumer to process the message.
However, both process all messages.
Moreover, when I start a new consumer, it does not care of current commited offset and process all messages in the stream again.

Did I miss something?

Thanks for your help.

Note that I use Symfony 4.1 messenger (so php-enqueue/messenger-adapter).

**Comment by makasim** [comment]
I think this should fix the issue https://github.com/php-enqueue/enqueue-dev/pull/508/files and [here](https://github.com/php-enqueue/enqueue-dev/issues/570#issuecomment-431428855) I proposed to back port the fix to 0.8 branch

**Comment by Gnucki** [comment]
Thanks!
